packages/ragbits-evaluate/src/ragbits/evaluate/dataset_generator/tasks/answer_gen_task.py

```diff
@@ -0,0 +1,50 @@
+from typing import Any
+
+from distilabel.steps.tasks import TextGeneration
+from ragbits.core.prompt.base import ChatFormat
+
+from ragbits.evaluate.dataset_generator.prompt_basic_answer_gen import BasicAnswerGenPrompt, BasicAnswerGenInput
+
+class AnswerGenTask(TextGeneration):
```

Comment:
i think that we can create abstract class for DistiLabel task - things like input_names, output_names, prompt_class, prompt_input class should be passed in constructor - the target class would only need to implement format_output method